specification/cognitiveservices/data-plane/ContentModerator/v1.0/ContentModerator.json

```diff
@@ -2865,114 +3400,397 @@
+                  "FrameImage": {
+                    "description": "Frame image Url.",
+                    "type": "string"
+                  },
+                  "ReviewerResultTags": {
+                    "type": "array",
+                    "items": {
+                      "required": [
+                        "Key",
+                        "Value"
+                      ],
+                      "type": "object",
+                      "properties": {
+                        "Key": {
+                          "description": "Your key parameter.",
+                          "type": "string"
+                        },
+                        "Value": {
+                          "description": "Your value parameter.",
+                          "type": "string"
+                        }
+                      }
+                    }
+                  },
+                  "Metadata": {
+                    "description": "Optional metadata details.",
+                    "type": "array",
+                    "items": {
+                      "required": [
+                        "Key",
+                        "Value"
+                      ],
+                      "type": "object",
+                      "properties": {
+                        "Key": {
+                          "description": "Your key parameter.",
+                          "type": "string"
+                        },
+                        "Value": {
+                          "description": "Your value parameter.",
+                          "type": "string"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+
+            },
+            "Metadata": {
+              "description": "Optional metadata details.",
+              "type": "array",
+              "items": {
+                "required": [
+                  "Key",
+                  "Value"
+                ],
+                "type": "object",
+                "properties": {
+                  "Key": {
+                    "description": "Your key parameter.",
+                    "type": "string"
+                  },
+                  "Value": {
+                    "description": "Your value parameter.",
+                    "type": "string"
+                  }
+                }
+              }
+            },
+            "Type": {
+              "description": "Type of the content.",
+              "enum": [
+                "Video"
+              ],
+              "type": "string"
+            },
+            "Content": {
+              "description": "Video content url to review.",
+              "type": "string"
+            },
+            "ContentId": {
+              "description": "Content Identifier.",
+              "type": "string"
+            },
+            "Status": {
+              "description": "Status of the video(Published,Unpublished,Pending)",
+              "type": "string"
```

Comment:
Can this be an enum?

Reply:
Created an enum status in the latest update.